oebDataFormats/.htaccess

```diff
@@ -0,0 +1,15 @@
+Options +FollowSymLinks
+
+Options -Indexes
+
+RewriteEngine on
+
+#Unversioned
+RewriteRule ^$ https://raw.githubusercontent.com/inab/OEB-ontologies/master/dataFormats.owl [R=302,L]
+
+#Latest version
+RewriteRule ^latest https://raw.githubusercontent.com/inab/OEB-ontologies/master/dataFormats.owl [R=302,L]
```

Comment:
FYI, You might consider using using a trailing `$` in these patterns, like `^latest$`.  Otherwise a request path like `/oebDataFormats/latestfoobar` would also match.  But maybe that's ok.  Same below.

Reply:
Yes, you are right. I have already corrected it. Thank you!